src/features/game/expansion/Land.tsx

```diff
@@ -336,9 +336,9 @@ const getIslandElements = ({
   mapPlacements.push(
     ...getKeys(chickens)
       // Only show placed chickens (V1 may have ones without coords)
-      .filter((index) => chickens[index]?.coordinates)
-      .flatMap((index, nameIndex) => {
-        const chicken = chickens[index]!;
+      .filter((id) => chickens[id]?.coordinates)
+      .flatMap((id, nameIndex) => {
```

Comment:
Because the `id` is a unique value we could scrap the `nameIndex` and just append the id for the component `key`.

```
key={`chicken-${id}`}
```